source/numerics.tex

```diff
@@ -17614,7 +17614,7 @@
 where the $i^\text{th}$ element is initialized to the result of
 \tcode{\placeholder{cmplx-func}(operator[]($i$))} for all $i$ in the range
 \range{0}{size()}, where \placeholder{cmplx-func} is the corresponding function
-from \tcode{<complex>}.
+from \libheader{<complex>}.
```

Comment:
\libheader takes an undecorated argument (no <>). Please check output of LaTeX.

Reply:
Yes, I already fixed it. Sorry for going too fast.